### ISSUE TITLE
Highlight alt text (or lack of) in development mode in the picture plugin

### ISSUE THREAD

**Issue by alexwlchan** [alexwlchan/alexwlchan.net#864]
I use the `JEKYLL_ENV` environment variable to distinguish between the local/live builds.

When doing local builds, it would be nice to make the alt text (or lack of) visible on the page, so I could review it as part of all the text on the page.

**Comment by alexwlchan** [comment]
I've got this working as a JavaScript snippet which gets embedded in development builds; I'm going to blog about it before I close this as done.
